commitlint.config.cjs

```diff
@@ -32,6 +32,7 @@ module.exports = {
         "popover",
         "notification",
         "table",
+        "split",
```

Comment:
Very well done! Let's just rename this as `split-button` here.

Reply:
Thanks. I changed it to **split-button**.